kolibri/plugins/coach/assets/src/views/common/assignments/IndividualLearnerSelector/index.vue

```diff
@@ -0,0 +1,130 @@
+<template>
```

Comment:
This is just a refactor to break down the current `IndividualLearnerSelector` component to have an `IndividualLearnerSelectorTable` that we can reuse in the `LearnersSelectorSidePanel`